src/App.tsx

```diff
@@ -1,25 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
```

Comment:
The `Pagination` component should be used with `total` and `perPage` props as specified in the task description. Currently, it is missing these props.